specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/databases.json

```diff
@@ -0,0 +1,1190 @@
+{
+  "swagger": "2.0",
```

Comment:
>swagger [](start = 3, length = 7)

Next time, please make the unmodified version of the previous API version swagger as the first commit. It makes diffing the delta much easier. (No action needed for this PR.)

Reply:
ok